extensions/org.wso2.carbon.config.maven.plugin/src/main/java/org/wso2/carbon/config/maven/plugin/ConfigDocumentMojo.java

```diff
@@ -189,14 +220,379 @@ private void writeConfigurationFile(Map<String, Object> finalMap, String filenam
             throw new MojoExecutionException("Error while creating new resource file from the classpath", e);
         }
 
-        // add configguration document to the project resources under config-docs/ directory.
+        // add configuration document to the project resources under config-docs/ directory.
+        Resource resource = new Resource();
+        resource.setDirectory(configDir.getAbsolutePath());
+        resource.setTargetPath(ConfigConstants.CONFIG_DIR);
+        project.addResource(resource);
+    }
+
+    /**
+     * read css file in the resource folder.
+     *
+     * @throws MojoExecutionException
+     */
+    private String readCSS() throws MojoExecutionException {
+        InputStream inputStream = ConfigDocumentMojo.class.getClassLoader().getResourceAsStream(CSS_FILE);
+        StringBuilder sb = new StringBuilder();
+
+        try (BufferedReader br = new BufferedReader(new InputStreamReader(inputStream, UTF_8_CHARSET))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                sb.append(line).append("\n");
+            }
+        } catch (IOException e) {
+            logger.error("Error while reading the css file.", e);
+        }
```

Comment:
InputStream not handled property. we need to close the stream in the finally block